great-idea/services.html

```diff
@@ -1,106 +1,100 @@
-<!doctype html>
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
 
-  <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
 
```

Comment:
without the lines above the browser wont read what you are saying correctly and that is why your services page is not working.